ingestion/src/metadata/ingestion/source/database/database_service.py

```diff
@@ -231,11 +231,20 @@ class DatabaseServiceSource(DBTMixin, TopologyRunnerMixin, Source, ABC):
     context = create_source_context(topology)
 
     def __init__(self):
-        dbt_details = get_dbt_details(self.source_config.dbtConfigSource)
-        self.dbt_catalog = dbt_details[0] if dbt_details else None
-        self.dbt_manifest = dbt_details[1] if dbt_details else None
-        self.dbt_run_results = dbt_details[2] if dbt_details else None
-        self.data_models = {}
+        if hasattr(self.source_config.dbtConfigSource, "dbtSecurityConfig"):
+            if self.source_config.dbtConfigSource.dbtSecurityConfig is None:
+                logger.warning("dbtConfigSource is not configured")
```

Comment:
nit - should this be a warning? It might be the usual case for many pipelines to not configure GCS/S3. IMO that could be just an INFO